styleguide/derek/global-components/social/_social.scss

```diff
@@ -1,8 +1,15 @@
+// sass-lint:disable no-ids no-important
+
 //Hack for Facebook to work.
 .fb-share-button {
   top: -8px;
 }
 
+//Hack for Google Plus.
+#___plus_0 {
+  width: 57px !important;
+}
```

Comment:
Since we control all the markup around these elements can we try something like this instead?

```html
<div class="social-button">
    <script type="IN/Share" data-url="rackspace.com"></script>
</div>
<div class="social-button">
    <div class="fb-share-button" data-href="<?php print $path; ?>" data-layout="button" data-size="small" data-mobile-iframe="true"><a class="fb-xfbml-parse-ignore" target="_blank" href="https://www.facebook.com/sharer/sharer.php?u=<?php print rawurlencode($path); ?>&amp;src=sdkpreparse"><?php print t('Share', [], ['context' => 'Social Buttons']); ?></a></div>
</div>
<div class="social-button">
    <a href="https://twitter.com/intent/tweet?text=<?php print $twitter_text; ?>" class="twitter-share-button" data-show-count="false"><?php print t('Tweet', [], ['context' => 'Social Buttons']); ?></a>
</div>
<div class="social-button">
    <div class="g-plus" data-action="share"></div>
</div>
<div class="social-button">
    <a class="social-shareEmail" href="mailto:?subject=<?php print $email_subject; ?>&body=<?php print $email_body; ?>" ><?php print t('Email', [], ['context' => 'Social Buttons']);?></a>
</div>
```

Then we can mandate with from an element above the actual social button. 

I'd just rather not use IDs or `!important` if possible. I did an initial test of this in the safari element inspector and it seems to work (although I admittedly haven't throughly tested it).